src/Components/CreateImageWizard/steps/Review/Footer/EditDropdown.tsx

```diff
@@ -22,19 +27,64 @@ type EditDropdownProps = {
   isDisabled: boolean;
 };
 
+const createAnalytics = (
+  requestBody: CreateBlueprintRequest,
+  packages: IBPackageWithRepositoryInfo[],
+  isBeta: () => boolean
+) => {
+  const analyticsData = {
+    image_name: requestBody.name,
+    description: requestBody.description,
+    distribution: requestBody.distribution,
+    openscap: requestBody.customizations.openscap,
+    image_request_types: requestBody.image_requests.map(
+      (req) => req.image_type
+    ),
+    image_request_architectures: requestBody.image_requests.map(
+      (req) => req.architecture
+    ),
+    imageRequests: requestBody.image_requests,
+    customizations: requestBody.customizations,
+    metadata: requestBody.metadata,
+    packages: packages.map((pkg) => pkg.name),
+    file_system_configuration: requestBody.customizations.filesystem,
+    module: AMPLITUDE_MODULE_NAME,
+    is_preview: isBeta(),
+    type: 'createBlueprintAndBuildImages',
```

Comment:
Do we want to use the same type as for `CreateDropdown`, or do we want to differentiate the actions? Maybe just something like:
```suggestion
    type: 'editBlueprintAndBuildImages',
```
Also is the addition of packages info we're sending to recommendations team in create action intentional here? Just curious, I bet they wouldn't mind getting more info about how are packages used :)

Reply:
The plan is not very specific, but I'd agree that it makes sense to differentiate the types, so I changed it :) As for the packages, they are intentional, they were mentioned in the doc connected to this task in Jira :memo: